cypress.json

```diff
@@ -10,7 +10,7 @@
       "react-dom": "node_modules/react-dom/umd/react-dom.development.js"
     },
     "cypress-react-selector": {
-      "root": "#cypress-root"
+      "root": "#cypress-jsdom"
```

Comment:
why if we use `<div id="cypress-root">` ?

Reply:
It is definitely strange, I just checked the HTML and found the 
![image](https://user-images.githubusercontent.com/16926049/92510025-df20f180-f213-11ea-800a-c2a72fec888c.png)

Reply:
Definitely, I had an outdated `dist` folder.